src/days-2-4-adv/adv.py

```diff
@@ -49,3 +64,69 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
+
+# initialize the player to be outside
+player = Player(room['outside'])
+
+#initialize a function to examine the room
+def look(room, player):
+    if len(room.items) == 0:
+        print("====================\n You don't see anything useful here. \n ====================")
+    if len(room.items) >= 1:
```

Comment:
Not sure it will actually make much difference in this particular instance but it would be better practice to use "elif" or "else". Truly though you can get rid of this second if altogether and the program should work fine and be a little cleaner.

Reply:
Good looking out, the second if clause is superfluous and has been removed.